packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -302,6 +305,45 @@ async function _getDashboardStats(app) {
     return false
   }
 
+  function _extractVmIdsFromBackupJob(job) {
+    let vmIds
+    try {
+      vmIds = extractIdsFromSimplePattern(job.vms).filter(vmId => app.hasObject(vmId, 'VM'))
```

Comment:
`extractIdsFromSimplePattern` will not throw an error if the pattern does not contains ids,  you should test if vmIds is undefined to switch to predicate/smart mode

Reply:
It will throw `Invalid pattern`, no?
https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/backups/extractIdsFromSimplePattern.mjs#L28